client/src/lib/dependency-injection/graphql/helpers.js

```diff
@@ -55,7 +58,14 @@ export const getFields = ({ args, fields, pagination = true }, stack = [ROOT_FIE
     const key = path.join('/');
     const fieldParams = args[key] || {};
 
-    const str = `${field}${getParams({ params: fieldParams, pagination: false })}`;
+    const items = (pagination) ? { ...fieldParams, ...paginationParams } : fieldParams;
+    const paramList = Object.entries(items)
+      .map(([paramName, varName]) => (
+        `${paramName}: $${varName}`
+      ));
+    const paramOutput = paramList.length ? `(${paramList.join(', ')})` : '';
+
+    const str = `${field}${paramOutput}`;
```

Comment:
Using `getParams()` here resulted in `lib/dependency-injection/tests/ApolloGraphqlManager-test.js` failing on its "adds args to fields" test.
Basically, an assumption had been made that `params` in the registered query would be an object of parameter name to variable name - but it should actually be parameter name to _input type_. This is backed up by the fact that `getVariables()` gets the variable _type_ and _name_ both from the params object.

I'm not 100% sure what the point of having variable names here is at all tbh... seems like it should just return the field names. But this makes the test happy without breaking any other tests, so it seems correct.